README.md

```diff
@@ -84,6 +84,76 @@ For supported exporters and their documentation see [exporters](/docs/exporters.
 
 For supported extensions and their documentation see [extensions](/docs/extensions.md).
 
+## Example
+
+Here is an example `config.yaml` setup for hostmetrics on Google Cloud. To make sure your environment is set up with required prerequisites, see our [Google Cloud Exporter Prerequisites](/config/google_cloud_exporter/README.md) page.
+
+```yaml
+receivers:
+  hostmetrics:
+    collection_interval: 60s
+    scrapers:
+      cpu:
+      disk:
+      load:
+      filesystem:
+      memory:
+      network:
+      paging:
+      processes:
+
+      # Uncomment "process" and "mute_process_name_error" to enable
+      # per process metrics for cpu, memory, and disk.
+      # This feature requires root / Administrative privileges.
+      #process:
+      #  mute_process_name_error: true
```

Comment:
I think it would be okay to remove this, as its just a quick example. The full example is available in `config/`

Reply:
I agree. Removed this section and added a line at the end of the example referencing full config.